CMakeLists.txt

```
@@ -22,6 +22,8 @@ set(BUILD_FUZZ OFF CACHE BOOL "Build bpftrace for fuzzing")
 set(USE_LIBFUZZER OFF CACHE BOOL "Use libfuzzer for fuzzing")
 set(FUZZ_TARGET "codegen" CACHE STRING "Fuzzing target")
 set(ENABLE_SYSTEMD OFF CACHE BOOL "Enable systemd integration")
+set(KERNEL_HEADERS_DIR "" CACHE PATH "Hard-code kernel headers directory")
+set(SYSTEM_INCLUDE_PATHS "auto" CACHE STRING "Hard-code system include paths (colon separated, auto queries clang at runtime)")
```

It took me a while to figure out the description, maybe a slight change like this would help?
```suggestion
set(SYSTEM_INCLUDE_PATHS "auto" CACHE STRING "Hard-code system include paths (colon separated, the default value \"auto\" queries clang at runtime)")
```

---

Sorry about that, reading back I agree it could read as "auto-queries clang.." which didn't make sense -- your suggestion looks good, applied!

---

Yep, that's exactly how my mind read it :-) Thanks!